flask_admin/static/admin/js/form-1.0.0.js

```diff
@@ -78,10 +78,6 @@
           return false;
         }
 
-        if (!window.MAPBOX_ACCESS_TOKEN) {
```

Comment:
If there's a way to check version on client-side and spit out warning if access token is missing?